src/Plugin.php

```diff
@@ -63,26 +76,11 @@ public function register_cli_command() {
 	 * @return void
 	 */
 	public function schedule_migration() {
-
-		$scheduler = new Migration\Migration_Scheduler();
-		if ( false === $scheduler->do_background_migration() || $scheduler->is_migration_scheduled() ) {
```

Comment:
This was the last remaining use of `do_background_migration()`, now that it's been removed, the method should be deleted from `Migration_Scheduler` (it's safe to delete it rather than deprecate it seeing as the plugin is still beta).